deploy/crds/kustomize_operatorpolicy/policy.open-cluster-management.io_operatorpolicies.yaml

```
@@ -221,52 +262,57 @@ spec:
                 - type
                 x-kubernetes-list-type: map
               relatedObjects:
-                description: List of resources processed by the policy
+                description: RelatedObjects reports a list of resources associated
+                  with the operator policy.
                 items:
-                  description: RelatedObject is the list of objects matched by this
-                    Policy resource.
+                  description: RelatedObject contains the details of an object matched
+                    by the configuration policy.
                   properties:
                     compliant:
+                      description: |-
+                        Compliant is whether the related object is compliant with the definition of the configuration
+                        policy.
```

Trying to get away from using the parameter name in the description. in understand that when policies are compliant, the object meets the configuration requirements
```suggestion
                      description: |-
                        Compliant represents whether the related object meets the defined conditions of the configuration
                        policy.
```